mne/epochs.py

```diff
@@ -978,6 +979,16 @@ def plot_topo_image(self, layout=None, sigma=0., vmin=None, vmax=None,
             border=border, fig_facecolor=fig_facecolor,
             fig_background=fig_background, font_color=font_color, show=show)
 
+    @copy_function_doc_to_method_doc(plot_compare_evokeds)
+    def plot_compare_evokeds(self, conditions=None, **kwargs):
```

Comment:
I wonder if, for the `Epochs` method, a clearer name might be `Epochs.plot_condition_averages()` or just `Epochs.plot_averages()`?